Classes/common/CDTReplicator/CDTAbstractReplication.m

```diff
@@ -19,6 +19,14 @@
 
 NSString *const CDTReplicationErrorDomain = @"CDTReplicationErrorDomain";
 
+@interface CDTAbstractReplication ()
+
+NS_ASSUME_NONNULL_BEGIN
+@property (nonnull, nonatomic, readwrite, strong) NSArray *httpInterceptors;
```

Comment:
Code style says to prefer [accessing properties over iVars](https://github.com/cloudant/CDTDatastore/blob/master/doc/style-guide.md#properties), and we use the `readwrite` nature [here](https://github.com/cloudant/CDTDatastore/pull/191/files#diff-c7f6d1c619815916886c1d6b3abc1be2R74). I could use the `addInterceptors` method, but I thought it would make sense to be consistent and access the property directly.